<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createCatalogWizard</class>
 <widget class="QWizard" name="createCatalogWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Catalog Wizard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QtRc.qrc">
    <normaloff>:/image/images/catalog_new.png</normaloff>:/image/images/catalog_new.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="newRadio">
      <property name="text">
       <string>New &amp;free catalog</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="fillRadio">
      <property name="text">
       <string>&amp;Create catalog from a directory</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="labelName">
      <property name="text">
       <string>N&amp;ame</string>
      </property>
      <property name="buddy">
       <cstring>lineName</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineName"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="labelTags">
      <property name="text">
       <string>Ta&amp;gs</string>
      </property>
      <property name="buddy">
       <cstring>lineTags</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineTags"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="labelDate">
      <property name="text">
       <string>&amp;Date</string>
      </property>
      <property name="buddy">
       <cstring>dateTimeEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDateTimeEdit" name="dateTimeEdit"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="labelDesc">
      <property name="text">
       <string>D&amp;escription</string>
      </property>
      <property name="buddy">
       <cstring>lineDesc</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="2" rowspan="2">
     <widget class="QTextEdit" name="lineDesc"/>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="labelChoseDir">
         <property name="text">
          <string>&amp;Please choose a directory for cataloging</string>
         </property>
         <property name="buddy">
          <cstring>lineDirPath</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineDirPath"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="buttonChoseDir">
         <property name="text">
          <string notr="true">...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>58</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage3">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="labelConf">
      <property name="text">
       <string>Your configuration:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="lineConf"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelReady">
      <property name="text">
       <string>If configuration is OK, so press the Next button.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage4">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelStat">
      <property name="text">
       <string>Operation finished.

Press Finish button.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>newRadio</tabstop>
  <tabstop>fillRadio</tabstop>
  <tabstop>lineName</tabstop>
  <tabstop>lineTags</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>lineDesc</tabstop>
  <tabstop>lineDirPath</tabstop>
  <tabstop>buttonChoseDir</tabstop>
  <tabstop>lineConf</tabstop>
 </tabstops>
 <resources>
  <include location="QtRc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fillRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
